### DISCUSSION TITLE
MPRIS Position Update

### DISCUSSION THREAD

**Discussion by ytzeng1** [mikebrady/shairport-sync#1769] [Ideas]
Could you implement the position in MPRIS interface? The progress estimation method in this old issue works well
https://github.com/mikebrady/shairport-sync/issues/1191#issuecomment-834734550

The main purpose would be able to use MPRIS lyrics softwares on top of shairport-sync. Thank you!

**Comment by mikebrady** [comment]
Thanks for the suggestion and the reminder. I’ll take a look at it.

**Comment by ytzeng1** [comment]
Thanks a lot! I’m looking forward to the update.On Dec 7, 2023, at 1:51 AM, Mike Brady ***@***.***> wrote:﻿
Thanks for the suggestion and the reminder. I’ll take a look at it.

**Comment by mikebrady** [comment]
Hi there. Still working on this, but just to say that IRL I'll be away from my computers until the end of January.

**Comment by ytzeng1** [comment]
Thanks for the update! On Jan 6, 2024, at 6:26 AM, Mike Brady ***@***.***> wrote:﻿
Hi there. Still working on this, but just to say that IRL I'll be away from my computers until the end of January.

**Comment by mikebrady** [comment]
Back at my computers now.

> Could you implement the position in MPRIS interface? The progress estimation method in this old issue works well https://github.com/mikebrady/shairport-sync/issues/1191#issuecomment-834734550

> The main purpose would be able to use MPRIS lyrics softwares on top of shairport-sync. Thank you!

This method can easily be added to the MPRIS interface, but it is non-MPRIS-standard. Please let me know if that would suit...
